src/work-allocation-2/containers/task-list-wrapper/task-list-wrapper.component.ts

```diff
@@ -2,7 +2,7 @@ import { ChangeDetectorRef, Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
 import { AlertService, LoadingService } from '@hmcts/ccd-case-ui-toolkit';
 import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
-import { Caseworker, Location } from 'api/workAllocation2/interfaces/task';
+import { Caseworker, Location } from 'api/workAllocation2/interfaces/common';
```

Comment:
Coming from `api/`, which it shouldn't.

Reply:
Changed to point to common interfaces in src folder.